docs/pages/docs/guides/workspaces.mdx

```diff
@@ -87,16 +85,15 @@ monorepo/
 ├─ sdk/
 ```
 
-Using the same workspace configuration, the `monorepo/docs/`, `monorepo/apps/api/`, `monorepo/apps/mobile/`, `monorepo/packages/config/`, and `monorepo/packages/tsconfig/` folders are all considered workspaces.
-The `monorepo/sdk/` folder is not considered a workspace and it is not included in the workspace configuration.
+In the example above, all directories inside `monorepo/apps/` and `monorepo/packages/` are workspaces, and the `monorepo/docs` directory itself is also a workspace. `monorepo/sdk/` is _not_ a workspace, as it is not included in the workspace configuration.
 
 ## Managing workspaces
 
-When you move, delete, or rename your workspaces, you will have to make sure that all folders linked within your `package.json` matches. Anytime you change the configuration of your workspace, make sure all the dependencies of the workspace are also updated. Re-run your npm client's install command to check your configuration. If there are any problems after that, you may have to delete your `node_modules` folder and run an install again.
+When you move, delete, or rename your workspaces, you will have to make sure that all folders linked within your `package.json` matches. Anytime you change the configuration of your workspace, make sure all the dependencies of the workspace are also updated. Re-run your package manager's install command to check your configuration. If there are any problems after that, you may have to delete your `node_modules` folder and run an install again.
 
 ## Managing dependencies
 
-To manage dependencies within workspaces in your monorepo, you will need to run commands that will only manage the dependencies of each workspace rather than for the entire monorepo.
+To manage dependencies within workspaces from the root of your monorepo, run commands that only manage the dependencies of each workspace rather than for the entire monorepo.
```

Comment:
```suggestion
To manage dependencies within workspaces from the root of your monorepo, run commands that only manage each workspace's dependencies rather than the entire monorepo.
```